src/registerToolbar.ts

```diff
@@ -57,6 +57,15 @@ export default function registerToolbar() {
         accelerators: ["p", "P"],
     });
 
+    ToolbarItemFactory.register({
```

Comment:
Let's also add a test to very the copy rect button renders in the toolbar since we have been bitten in the past where buttons were accidentally removed.

Reply:
Still not seeing this.